infrastructures/docker/infrastructure/src/main/java/org/eclipse/che/workspace/infrastructure/docker/server/mapping/ServersMapper.java

```diff
@@ -155,8 +171,26 @@ public ServersMapper(String hostname, String machineName) {
    * |------------------------------------------------------------------------|
    * </pre>
    */
+  private String makeUrl(ContainerPort port, ServerConfig cfg, String hostname)
```

Comment:
We have only one usage of a method with name `makeUrl` in the class. We don't need 2 methods for that.